notebook/Makefile

```diff
@@ -8,7 +8,7 @@ PUSHED_LATEST_TAG = $(foreach image,$(IMAGE_SHORT_NAMES),build/images/$(image)/_
 
 .PHONY: $(IMAGE_SHORT_NAMES)
 
-build/images/%/_BUILT: images/%/Dockerfile $(shell find images/$*)
+build/images/%/_BUILT: images/%/Dockerfile $(shell find images/$%*)
```

Comment:
what does `$%*` evaluate to? I'm not familiar with that make variable.

Reply:
$% is the % bit. The * is just a *

Reply:
oh, hmm, maybe not